ch_backup/cli.py

```diff
@@ -100,6 +122,17 @@ def signal_handler(signum, _frame):
     help="Disable certificate verification for https protocol.",
 )
 @option("--zk-hosts", type=str, help="Use specified zk hosts for connection to the ZK")
+@option(
+    "--config-value",
```

Comment:
Suggest to change to  `"--parameter", "--config-parameter"`. I think it will be more intuitive clear cli option names.

Reply:
I like "--config-parameter" more, because it's name is correlated with "config" parameter.

Btw, @Alex-Burmak, what do you think about breaking changes: remove parameters `--host/--protocol/...` from cli?
https://github.com/yandex/ch-backup/blob/main/ch_backup/cli.py#L89
Because now we can use current logic to define them.

Fix it now or take some time to test "--config-parameter"?

Reply:
I think it's OK to have the both new universal `--config-parameter` and existing specialized options. `--port`, `--protocol`, etc. are options for most common scenarios. `--config-parameter` option is for advanced configuration / scenarios.